### PR TITLE
WIP Aga Branch

### PR DESCRIPTION
**Pull request by AdamENESS**
Kain, have a play. you can create 128/256 Colour Screens.

Still need to implement the KILLEHB flag for 64 colours.

### CODE REVIEW

**Review by tehKaiN — CHANGES_REQUESTED** [review]
Looks generally good, but needs some work:

- please fix the formatting to conform to codestyle
- please fix problems stated in the comments

after that and killing EHB, I think we're ready for merge.

I still don't know what to think about storing OCS/AGA palettes separately, but I have no better idea atm

**Comment by AdamENESS** [comment]
> Looks generally good, but needs some work:
> 
> * please fix the formatting to conform to codestyle
> * please fix problems stated in the comments
> 
> after that and killing EHB, I think we're ready for merge.
> 
> I still don't know what to think about storing OCS/AGA palettes separately, but I have no better idea atm

It really annoys me too tbh, but I can't think of a better way, other then with some dynamic allocation depending on the flags set, and then dirty casts :/

File: src/ace/utils/extview.c [hunk 1]
```diff
@@ -20,28 +21,35 @@ tView *viewCreate(void *pTags, ...) {
...
-	else {
+	else
+	{
 		pView->pCopList = copListCreate(0, TAG_DONE);
 	}
 
 	// Additional CLUT tags
-	if(tagGet(pTags, vaTags, TAG_VIEW_GLOBAL_CLUT, 0)) {
+	if (tagGet(pTags, vaTags, TAG_VIEW_GLOBAL_CLUT, 0))
+	{
```
[the remark this answers is not in the document]
Comment (AdamENESS) [reply src/ace/utils/extview.c:41]: I think that is how I have VSCode set to do autoformatting, I shall look at changing those settings..

[the remark this answers is not in the document]
Comment (tehKaiN) [reply src/ace/utils/extview.c:41]: I don't think you'll manage to fit those settings to use ACE's codestile (especially `while(...){}` busyloops). Maybe just use workspace setings to disable autoreformatting if you have that set, or if you call "Format document" option manually, then don't ;)

**Pull request closed without merging**